model/Licensing/Licensing.md

```diff
@@ -8,6 +8,8 @@ Additional metadata relating to software licensing.
 
 ## Description
 
+TODO: This text has to be reworked to reflect the new structure.
+
 The Licensing namespace defines metadata fields applicable to software
 licensing. It also defines the classes and properties that comprise the
 SPDX License Expression syntax and that relate to the SPDX License List.
```

Comment:
This makes me think we should rename `ExpandedLicense` to `Licensing` with the same description above and rename the current `Licensing` to `SimpleLicensing`.

Reply:
> Maybe it will be clearer to have SimpleLicensing and ExpandedLicensing profiles.

I like this naming.  In `SimpleLicensing` we could refer to `ExpandedLicensing` for the detailed definitions.

I would suggest waiting on any renaming until @swinslow has a chance to weigh in.

Reply:
I implemented the renaming.